Sources/XCTest/XCTAssert.swift

```diff
@@ -52,25 +52,19 @@ private enum _XCTAssertionResult {
     case UnexpectedFailure(ErrorProtocol)
 
     var expected: Bool {
-        switch (self) {
-        case .UnexpectedFailure(_):
-            return false
-        default:
-            return true
+        switch self {
+        case .UnexpectedFailure(_): return false
```

Comment:
In #79, you made changes that placed a line break after a `case` statement. Here you've removed line breaks. Do you have a clear goal in mind here?